src/navigations/root-stack.js

```diff
@@ -67,6 +68,7 @@ export const RootNavigator = () => {
   useLocalDatabaseHook(true);
 
   React.useEffect(() => {
+    StatusBar.setBarStyle('light-content', true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `StatusBar.setBarStyle('light-content', true);` line has been added. This sets the text and icons in the status bar to light content for better visibility on a dark background. It's a good addition considering the color scheme changes.

```diff
+ StatusBar.setBarStyle('light-content', true);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->